react/tic-tac-toe/src/redux/auth/reducer.js

```diff
@@ -4,9 +4,10 @@ import { actions } from './actions';
 
 const initialState = {
   values: {},
```

Comment:
Delete this

Reply:
Already made this correction on another branch.